jvm/benchmarks/Main.hs

```diff
@@ -116,6 +116,14 @@ benchRefs =
     ,  bench "global reference (no finalizer)" $ nfIO $ do
         _ <- newGlobalRefNonFinalized jobj
         return ()
+    , bench "delete global reference (no finalizer)" $ nfIO $ do
+        ref <- newGlobalRefNonFinalized jobj
+        _ <- deleteGlobalRefNonFinalized ref
+        return ()
+    , bench "delete global reference (no finalizer, safe)" $ nfIO $ do
+        ref <- newGlobalRefNonFinalized jobj
+        _ <- runInAttachedThread (deleteGlobalRefNonFinalized ref)
```

Comment:
`runInAttachedThread` does not attach the thread if the thread is already attached. You would have to try running this benchmark on a different thread.